src/server/transaction/pretty/pretty.go

```diff
@@ -0,0 +1,178 @@
+package pretty
+
+import (
+	"fmt"
+	"html/template"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/pachyderm/pachyderm/src/client/pfs"
+	"github.com/pachyderm/pachyderm/src/client/transaction"
+	"github.com/pachyderm/pachyderm/src/server/pkg/pretty"
+)
+
+const (
+	//TransactionHeader is the header for transactions.
+	TransactionHeader = "TRANSACTION\tSTARTED\tLENGTH\t\n"
```

Comment:
What would you think of renaming `LENGTH` to `OPS` or something like that? My immediate thought when I read "LENGTH" was that it was about the length of time the txn had been running for, (which it pretty obviously isn't, since that's what `STARTED` is), but I still think it might be a bit too generic of a name.

Reply:
Absolutely - I flip-flopped on this a bit before landing on length, but `OPS` sounds much better.